standards/lotus-family/conformance/lotus_family_system_model.py

```
@@ -0,0 +1,151 @@
+        raise ValueError("unsupported route schema")
+    view = validate_graph(graph)
+    routes = rows(route_model, "routes", ROUTE_COLS)
+    route_ids: set[str] = set()
+    for route in routes:
+        if route["id"] in route_ids:
+            raise ValueError(f"duplicate route: {route['id']}")
+        route_ids.add(route["id"])
+        path = route["path"]
+        if len(path) < 2:
+            raise ValueError(f"route too short: {route['id']}")
+        for pair in zip(path, path[1:]):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)